gradle.properties

```
@@ -17,9 +17,11 @@ POM_URL=https://github.com/requery/sqlite-android/
 POM_SCM_URL=https://github.com/requery/sqlite-android/
 POM_SCM_CONNECTION=scm:git:git://github.com/requery/sqlite-android.git
 POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/requery/sqlite-android.git
+POM_LICENCE_NAME=The Apache Software License, Version 2.0
+POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
```

```suggestion
POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0
```

I don't think there's a practical difference here, but I'm a bit of a golfer and that file extension annoys me 😂